app/src/pages/inside/projectSettingsPageContainer/content/patternAnalysis/modals/editPatternModal/editPatternModal.jsx

```diff
@@ -77,7 +77,12 @@ const EditPatternModal = ({ data, handleSubmit, initialize, dirty }) => {
       allowCloseOutside={!dirty}
     >
       <div className={cx('description')}>{formatMessage(messages.editPatternName)}</div>
-      <FieldElement name="name" label={formatMessage(messages.patternName)} isRequired>
+      <FieldElement
+        name="name"
+        label={formatMessage(messages.patternName)}
+        isRequired
+        dataAutomationId="patternNameField"
```

Comment:
As I can see, the FiledText component does nothing with the dataAutomationId prop.